app/controllers/content_item_signups_controller.rb

```diff
@@ -18,7 +18,14 @@ def new
     end
   end
 
-  def confirm; end
+  def confirm
+    if is_taxon?(@content_item) && taxon_children(@content_item).any?
```

Comment:
worth extracting this as `is_taxon_with_children?` ? I note it's used in a couple of places now, and it'll ensure that the logic stays in step

Reply:
Sure, that makes sense. I've added another commit for that 👍.